app/views/state_file/questions/ny_primary_state_id/_state_id.html.erb

```diff
@@ -40,10 +40,12 @@
           <span id="checkbox">
             <%= f.cfa_checkbox(:non_expiring, t("state_file.questions.primary_state_id.state_id.id_details.no_expiration_date")) %>
           </span>
-          <%= f.cfa_select(:state, t("state_file.questions.primary_state_id.state_id.id_details.issue_state"), States.name_value_pairs, include_blank: true, classes: ["form-width--short, spacing-above-0"]) %>
-          <% if options[:include_first_three_doc_num] == true %>
+          <div onchange="$('#first_3_doc_num_container')[event.target.value == 'NY' ? 'show' : 'hide']()">
```

Comment:
The requirement is that the 3 doc number only shows if the license selection is New York. We don't currently have a pre-built component for showing / hiding a section based on a select, so some custom JS was needed.